site/server/bakeChartsToImages.tsx

```diff
@@ -26,13 +26,16 @@ export async function getChartsAndRedirectsBySlug() {
     return chartsBySlug
 }
 
-export async function getChartsBySlug() {
+export async function getChartsBySlug(onlyPublished = true) {
```

Comment:
Nit: if we don't ever actually want to get the drafts, should we just rename this method to `getPublishedChartsBySlug` and remove the parameter entirely? One less code path to test :)

Reply:
Great idea actually, I've just changed that.